src/OnboardingSPA/components/AdminBarSiteGen/index.js

```diff
@@ -0,0 +1,30 @@
+import { __ } from '@wordpress/i18n';
+import { Icon, wordpress } from '@wordpress/icons';
+
+const AdminBarSiteGen = ( {} ) => {
+	return (
+		<div className="nfd-sitegen-admin-bar">
+			<div className="nfd-admin-bar-wplogo">
+				<Icon icon={ wordpress } />
+				<span>{ __( 'WordPress', 'wp-module-onboarding' ) }</span>
+			</div>
+			<div className="nfd-sitegen-admin-bar-profile">
+				<span className="nfd-admin-bar-greeting">
+					Howdy, Maya Jiménez
+				</span>
+				<img
+					alt=""
+					src="http://1.gravatar.com/avatar/729ae85bf62b9917e93538db2f2688ca?s=26&amp;d=mm&amp;r=g"
+					srcSet="http://1.gravatar.com/avatar/729ae85bf62b9917e93538db2f2688ca?s=52&amp;d=mm&amp;r=g 2x"
```

Comment:
Let's not add images like this might be possible they might even break in the future maybe you can see how we are doing it rn [Link](https://github.com/newfold-labs/wp-module-onboarding/blob/f294739dc152a2ec1e64eb70bd465156933a78b3/src/OnboardingSPA/styles/_icons.scss#L20C2-L22C65)